src/components/FileDetail/UploadVersionModalContainer.js

```diff
@@ -0,0 +1,127 @@
+import React, {useState} from 'react';
+import {Query, Mutation} from 'react-apollo';
+import {GET_FILE_BY_ID} from '../../state/queries';
+import {CREATE_FILE} from '../../state/mutations';
+import {LoadingPlaceholder} from '../Loading';
+import FileUploadTarget from '../FileUpload/FileUploadTarget';
+import Modal from '../Modal/Modal';
+
+const UploadVersionModalContainer = ({
```

Comment:
This could be moved to `src/containers`

Reply:
would really like to reorg the folder structure somehow. I feel like the container concept isn't scaling the best or maybe I'm going about it wrong